src/models/fixtures/city.js

```diff
@@ -1,16 +1,11 @@
 import fixture from 'can-fixture';
+import City from '../city';
 
 const store = fixture.store([
   { state: 'CA', name: 'Casadina' },
   { state: 'NT', name: 'Alberny' }
-],{});
+], City.connection.algebra);
```

Comment:
@justinbmeyer this will stay as `City.connection.algebra` since we're just allowing superMap to set it, right?